packages/react-native/Libraries/StyleSheet/splitLayoutProps.js

```diff
@@ -49,6 +49,7 @@ export default function splitLayoutProps(props: ?____ViewStyle_Internal): {
         case 'bottom':
         case 'top':
         case 'transform':
+        case 'transformOrigin':
```

Comment:
What is this function used for? Neither `transform` or `transformOrigin` are layout props in the strict sense of the term.

Reply:
My bad. I added without thinking here as I saw transform. Seems transform is added to handle refresh control for the inverted scroll view. https://github.com/facebook/react-native/commit/0a282c42b4d1c2316513cd5588a0a92b54db2991. transform-origin here doesn't make much sense, so removing.